src/projectile.cpp

```diff
@@ -0,0 +1,221 @@
+#include "projectile.h"
+
+#include "game.h"
+#include "map.h"
+#include "map_iterator.h"
+#include "explosion.h"
+#include "field.h"
+
+projectile::projectile() :
```

Comment:
Why? It's just as good here and will result in the same code anyway. And putting it in the header exposes an implementation detail - it's nobodies business what the initial value of private members is.